partner_second_lastname/models/res_config_settings.py

```diff
@@ -33,3 +36,18 @@ def _partners_for_recalculating(self):
                 ("lastname2", "!=", False),
             ]
         )
+
+    def action_recalculate_partners_name(self):
+        # Override of partner_firstname's action_recalculate_partners_name
+        # to avoid triggering _inverse_name when pressing Recalculate Names button
+        self.env["ir.config_parameter"].sudo().set_param(
+            "partner_names_order", self.partner_names_order
+        )
+        partners = self._partners_for_recalculating()
+        _logger.info("Recalculating names for %d partners.", len(partners))
+        self.env.add_to_compute(self.env["res.partner"]._fields["name"], partners)
```

Comment:
this is the main change right? 

why not doing it in original method definition?

Reply:
I was trying to keep this change and the unittest in the same module, but since partner_second_lastname depends on partner_firstname, since there will be no dependency problems, so there's no reason to override after all, thank you!